rts/lra/test/pom.xml

```diff
@@ -52,14 +67,154 @@
+                        <project.build.directory>${project.build.directory}</project.build.directory>
+                    </systemPropertyVariables>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
     
     <profiles>
+        <profile>
+            <!-- when arq profile is activated we expect to start coordinator -->
+            <id>arq</id>
+            <properties>
+                <!-- making the Quarkus coordinator to be started -->
+                <lra.coordinator.exec.plugin.phase>pre-integration-test</lra.coordinator.exec.plugin.phase>
+                <!-- where the Thorntail test application will be started at (the Quarkus coordinator goes at 8080) -->
+                <test.application.port>8180</test.application.port>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>io.thorntail</groupId>
+                    <artifactId>arquillian</artifactId>
+                    <scope>test</scope>
+                </dependency>
+                <dependency>
+                    <groupId>io.thorntail</groupId>
+                    <artifactId>cdi</artifactId>
+                    <version>${version.thorntail}</version>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <skipITs>false</skipITs>
+                            <systemProperties>
+                                <arquillian.xml>arquillian-thorntail.xml</arquillian.xml>
+                            </systemProperties>
+                            <systemPropertyVariables>
+                                <thorntail.test.logging.category.arjuna>${thorntail.test.logging.category.arjuna}</thorntail.test.logging.category.arjuna>
+                                <thorntail.test.logging.category.narayana>${thorntail.test.logging.category.narayana}</thorntail.test.logging.category.narayana>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>arq.wildfly</id>
+            <properties>
+                <!-- the AppServerCoordinatorDeploymentObserver creates the deployment named 'lra-coordinator' and deploys it to the app server,
+                     the last part of the path with value 'lra-coordinator' is defined by LRA coordinator JAX-RS endpoint -->
+                <lra.coordinator.url>http://${lra.coordinator.host}:${lra.coordinator.port}/lra-coordinator/lra-coordinator</lra.coordinator.url>
```

Comment:
maybe you can change the name of the deployment so it's different than lra-coordinator? not sure if this can be confusing for anyone.

Reply:
I have the same comment as above in point 2, let's keep it consistent. If http://localhost:8080/lra-coordinator works then use that. If the endpoint keeps changing depending upon which test is being ran or debugged then it increases the number of surprises that the developer has to deal with and, again, it makes working with the code base a less pleasant experience.

Reply:
Ok, let's go with what is it defined now.

I'm resolving this discussion. If there is something that should be adjusted and I missed that, please reopen it.